client/src/Components/sitterProfile/bookSitter/BookSitter.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { TextField, Button, Paper } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { KeyboardTimePicker, KeyboardDatePicker } from "@material-ui/pickers";
+import "./BookSitter.css";
+// The first commit of Material-UI
+const useStyles = makeStyles({
+  bookSitterPaper: {
+    maxWidth: "400px",
+    textAlign: "center",    
+  },
+  picker: {
+    maxWidth: "130px",
+  },
+  bookButton: {
+    background: "tomato",
```

Comment:
i see we're using a mix of useStyles and .css files. we probably want to stay consistent with these things!

Reply:
yeah i wanted to use css grid but I couldn't find a way to do that in use styles, and if i wanna style mui components i cant put that in the css file. every time i search up using css grid in usestyles muis grid stuff always came up.